addons/sms/static/src/components/sms_widget/fields_sms_widget.js

```diff
@@ -88,10 +89,10 @@ export class SmsWidget extends EmojisTextField {
      * @private
      */
     async onInput(ev) {
-        await this.props.update(this.targetEditElement.el.value);
         super.onInput(...arguments);
+        await this.props.update(this.targetEditElement.el.value);
     }
 }
 SmsWidget.template = 'sms.SmsWidget';
-SmsWidget.additionalClasses = [...(EmojisTextField.additionalClasses || []), 'o_field_text'];
+SmsWidget.additionalClasses = [...(EmojisTextField.additionalClasses || []), 'o_field_text', 'o_field_text_emojis'];
```

Comment:
As discussed, would be nice to find where this "o_field_text_emojis" class is added in the DOM.

It looks like it's some kind of automatic magic based on the field / component, but which one?
And also, if it's magically added, why do we specify here?

EDIT: Looks like there is also a "o_field_char_emojis", so it's probably based on the field name.
EDIT again: ok, solved, it's based on the widget name.